src/ripple/app/misc/HashRouter.h

```diff
@@ -119,13 +119,14 @@ class HashRouter
     // VFALCO NOTE this preferred alternative to default parameters makes
     //         behavior clear.
     //
-    static inline int getDefaultHoldTime ()
+    static inline std::chrono::seconds getDefaultHoldTime ()
     {
-        return 300;
+        return std::chrono::seconds(300);
     }
 
-    explicit HashRouter (int entryHoldTimeInSeconds)
+    explicit HashRouter (Stopwatch& clock, std::chrono::seconds entryHoldTimeInSeconds)
         : mHoldTime (entryHoldTimeInSeconds)
+        , mSuppressionMap(clock)
```

Comment:
Thanks for spotting that. Fixed with a simple reordering. Interesting that never showed up before, maybe because `mHoldTime` used to be a simple `int`.